src/displayapp/widgets/Counter.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <lvgl/lvgl.h>
+
+namespace Pinetime {
+  namespace Applications {
+    namespace Widgets {
+      class Counter {
+      public:
+        Counter(int min, int max);
```

Comment:
if the counter can only hold values with 2 digits. Then we could change this to `int8_t`. If we further assume to not have negative numbers we could use `uint8_t`

Reply:
The Counter can easily be made to hold larger values if there's a use for it. It looks like using int actually produces a tiny bit smaller binary. Would there be benefits to changing it?

Reply:
int makes is smaller? that's unexpected. It would have just been more explicit and matching the value range the counter is designed for better

Reply:
I think it's because int is the native integer size of the processor. This may be something worth looking into elsewhere as well. There's int_fast8_t for example, which may use a larger size if it is faster, and in this case using it would produce the same sized binary, assuming it uses the native size.